assets/constants.ts

```diff
@@ -186,31 +186,31 @@ export = {
     "description": "فارسی (Persian)"
   }, {
     "code": "pl",
-    "description": "polski (Polish)"
+    "description": "Polski (Polish)"
   }, {
     "code": "pt",
-    "description": "português (Portuguese)"
+    "description": "Português (Portuguese)"
   }, {
     "code": "ro",
-    "description": "română (Romanian)"
+    "description": "Română (Romanian)"
   }, {
     "code": "ru",
-    "description": "русский (Russian)"
+    "description": "Pусский (Russian)"
   }, {
     "code": "sr",
-    "description": "српски (Serbian)"
+    "description": "Cрпски (Serbian)"
   }, {
     "code": "sk",
-    "description": "slovenčina (Slovak)"
+    "description": "Slovenčina (Slovak)"
```

Comment:
here it should be `"slovenčina (Slovak)"` (https://sk.wiktionary.org/wiki/sloven%C4%8Dina)